docs/managing-findings.md

```diff
@@ -0,0 +1,108 @@
+1:print(1 == 1)
+--------------------------------------------------------------------------------
+severity:warning rule:finding-test: Finding test 2
+1:print(1 == 1)
+```
+
+*For more information on writing rules, see [Rule syntax](writing-rules/rule-syntax.md).*
+
+## Semgrep Action
+
+Semgrep Action builds on Semgrep CLI findings to track the lifetime of an individual finding.
+
+A Semgrep Action finding is defined by a 4-tuple:
+
+```
+(rule ID, file path, syntactic context, index)
+```
+
+These pieces of state correspond to:
+
+1. `rule ID`: the rule's ID within the Semgrep ecosystem.
+1. `file path`: the filesystem path where the finding occurred.
+1. `syntactic context`: the lines of code corresponding to the finding.
+1. `index`: an index into identical findings within a file. This is used to disambiguate findings.
+
+!!! note
+    `syntactic context` is normalized by removing indentation, [`nosemgrep`](ignoring-findings.md#ignoring-findings-via-inline-comments) comments, and whitespace.
+
+These 4 pieces of information are then hashed and returned as the syntactic identifier: `syntactic_id`. This is how Semgrep Action uniquely identifies findings and tracks them across state transitions. Semgrep Action does not store or transmit code contents. The `syntactic context` is hashed using a one-way hashing function making it impossible to recover the original contents.
+
+## Semgrep App
+
+Semgrep App builds on Semgrep Action findings to track state transitions and provide additional context for managing findings within your organization. Findings move between states according to their Semgrep Action `syntactic_id`, as mentioned above. A finding can occupy 3 states in Semgrep App: `OPEN`, `FIXED`, and `MUTED`.
+
+### Finding states
+
+Semgrep App finding states are defined as follows:
+
+1. `OPEN`: the finding exists in the code and has not been muted.
+1. `FIXED`: the finding existed in the code, and is no longer found.
+1. `MUTED`: the finding has been ignored by a `nosemgrep` comment.
+
+Findings move between states as follows:
+
+![Finding state transitions](img/finding-state.png "Finding state transitions")
+
+These transitions are defined as follows:
+
+1. `Fix`: a previously identified `syntactic_id` no longer exists.
+1. `Regression`: a previously fixed `syntactic_id` has been reintroduced.
+1. `Mute`: a previously identified `syntactic_id` has been ignored.
+1. `Unmute`: a previously muted `syntactic_id` has been unignored.
+
+### Analytics
+
+Semgrep App provides analytics to measure Semgrep performance within your organization. Visit [Manage > Analytics](https://semgrep.dev/manage/findings) and use measurements like fix rate and findings over time to get the most out of your Semgrep deployment:
+
+![Blocking vs. non-blocking findings](img/sankey-diagram.png "Blocking vs. non-blocking findings")
+
+Filter findings to drill down into specific areas:
+
+![Findings filters](img/findings-filters.png "Findings filters")
```

Comment:
These last three screenshots are bit hard to read. Any chance you can make their text larger? Perhaps chop off some of the whitespace on their right sides?

Reply:
Are you sure it's not just the image in the Github PR description? This seems reasonable to me: https://github.com/returntocorp/semgrep-docs/blob/3195425b43dafb8affbb55740aff808373f1f0fc/docs/img/ruleset-findings.png

The findings filters image is definitely lopsided, should I just update that one?

Reply:
I think it’s not just the image in the PR description because I checked out your repo when reviewing the changes and looked at them on the built site. Here’s what I see:
![Screen Shot 2021-02-09 at 12 38 21 PM](https://user-images.githubusercontent.com/302941/107425445-d50d6280-6ad3-11eb-90c5-74ae6488d2e9.png)

And now the findings filters image is much more easy to read than it was.

Reply:
Tightened up the screenshots as much as I could :+1: